### ISSUE TITLE
Add new class <Observational_Intent>

### ISSUE THREAD

**Issue by thareUSGS** [pds-data-dictionaries/ldd-msn_surface#4]
Per Bob. D. (for M2020).

Observational_Intent = Describes the intent and context of the observation
```xml
<Observational_Intent>
       <campaign>Identifies the long-term observational campaign of which this observation is a part</campaign>
       <goal>Identifies the goal of this observation</goal>
       <task>Identifies the specific task for this observation</task>
       <activity_notes>Provides additional notes regarding the activity</activity_notes>
</Observational_Intent>
```

**Comment by rgdeen** [comment]
the above lost some info.  The keywords are <campaign>, <goal>, <task>, and <activity>

**Comment by rgdeen** [comment]
omg!!  git is eating the text.  Okay without the brackets... campaign, goal, task, activity_notes

**Comment by thareUSGS** [comment]
@rgdeen how does that look above - fixed github eating the alligators.

**Comment by rgdeen** [comment]
yeah that looks right

**Comment by rgdeen** [comment]
According to the 2020 folks there should be three keywords here:

```
  <goal> Identifies the goal of this observation
  <task> Identifies the specific task for this observation
  <keyword>Identifies geological characteristics of a feature or target
```
I'm double checking to make sure they really don't want <campaign>... kinda thinking we could include it anyway.

**Comment by rgdeen** [comment]
I see you just added the others so maybe leave them there and just add the <keyword> one.

**Comment by thareUSGS** [comment]
@rgdeen want to review this?
(1) Everything was set to 0 or 1 (not unbounded) 
https://github.com/pds-data-dictionaries/ldd-msn_surface/blob/996fab53e4b0611ddd722202ce41e9822205f4c8/src/PDS4_MSN_SURFACE_IngestLDD.xml#L699

including Observational_Intent
https://github.com/pds-data-dictionaries/ldd-msn_surface/blob/996fab53e4b0611ddd722202ce41e9822205f4c8/src/PDS4_MSN_SURFACE_IngestLDD.xml#L496

(2) attributes are all ASCII_Text_Preserved (long string allowed)
https://github.com/pds-data-dictionaries/ldd-msn_surface/blob/996fab53e4b0611ddd722202ce41e9822205f4c8/src/PDS4_MSN_SURFACE_IngestLDD.xml#L432

**Comment by rgdeen** [comment]
it says Observational_Intent is an attribute_of SMI... but the others say component_of.  It's a class not an attribute so should it be component_of?

Looks good otherwise, except to also add <keyword>

**Comment by thareUSGS** [comment]
Good catch on component_of. That is what it should be.

"keyword" I can add, but sort-of a generic name for "geological" characteristic. Does it make sense to use something more apparent like
"geologic_setting"
"environment"
"site_description"
....

**Comment by rgdeen** [comment]
If you're asking me, yeah... but that's apparently what 2020 is going with.

If you want to wait on this for things to settle out (I did send a reply to clarify what the names are) you can... this is not something that's needed for MSAM, it's looking forward to 2020.  So we could go for now with what you've already done and then add keyword or whatever later.

**Comment by rgdeen** [comment]
just either keep this issue open or close it and add a new one for just keyword so we don't forget.

**Comment by thareUSGS** [comment]
added "keyword" (for geologic characteristics) and published msn_surface v1200 for pre-release.
